src/table_builder.cc

```diff
@@ -266,5 +261,29 @@ void TitanTableBuilder::UpdateInternalOpStats() {
   }
 }
 
+Status TitanTableBuilder::GetBlobRecord(const BlobIndex& index,
+                                        BlobRecord* record) {
+  auto storage = blob_storage_.lock();
```

Comment:
no need to call it everytime

Reply:
How. Is it safe to hold a shared_ptr in TitanTableBuilder?